Dockerfile

```
@@ -0,0 +1,29 @@
+# syntax=docker/dockerfile:experimental
+
+FROM python:3.12.9-alpine as builder
+
+# create virtual environment
+RUN python -m venv /opt/venv
+ENV PATH="/opt/venv/bin:$PATH"
+
+COPY requirements.txt .
```

Hadolint (2.12.0)

[warning] `COPY` to a relative destination without `WORKDIR` set.

(DL3045)